### PR TITLE
Cass changelog blog 2 update

### PR DESCRIPTION
**Pull request by loganloganlogan**
@ossarga Made the updates as requested.

### CODE REVIEW

**Comment by ossarga** [comment]
LGTM. Post renders correctly. Let's ship it.

File: src/_posts/2020-12-04-cass_changelog_2.markdown [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "Apache Cassandra Changelog #2 | December 2020"
+date:   2020-12-01 x
```
Comment (michaelsembwever) [inline src/_posts/2020-12-04-cass_changelog_2.markdown:4]: @ossarga what's with the `x` here?

Comment (michaelsembwever) [reply src/_posts/2020-12-04-cass_changelog_2.markdown:4]: fixed.

**Comment by michaelsembwever** [comment]
Merged to master.
https://github.com/apache/cassandra-website/commit/5b5a495b8cdfeb7f35d80c0170308c1fabd77ada

**Pull request closed without merging**